packages/database/src/migrations/20210712082624-AddAdminPanelSessionTable-modifies-schema.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.createTable('admin_panel_session', {
```

Comment:
Could you please create a card for storing the session outside of the postgres database? We could use redis or some other session cache, rather than having all these extra dependencies on the central tupaia db

Reply:
Created: https://github.com/beyondessential/tupaia-backlog/issues/3188